k8s/helm-deploy.sh

```diff
@@ -27,7 +27,7 @@ helm upgrade --install $BRANCH . \
      -f values-prod-config.yaml \
      --set studioApp.imageName=gcr.io/$PROJECT_ID/learningequality-studio-app:$COMMIT \
      --set studioNginx.imageName=gcr.io/$PROJECT_ID/learningequality-studio-nginx:$COMMIT \
-     --set studioApp.releaseCommit=$COMMIT} \
+     --set studioApp.releaseCommit=$COMMIT \
```

Comment:
Oh, this extra commit got deployed. This is the PR for it: https://github.com/learningequality/studio/pull/917